api-extractor.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "<projectFolder>/resources/embedding-sdk/dist/enterprise/frontend/src/embedding-sdk/index.d.ts",
```

Comment:
Is it possible put this `api-extractor.json` config under the `enterprise/frontend` as it is only used for the Embedding SDK? Just so we don't pollute the project's top-level directory.

Reply:
let me check

Reply:
Thanks! If possible, would be awesome to rename the config file too (e.g. `embedding-sdk-api-extractor.json`)

Reply:
moved as `enterprise/frontend/src/embedding-sdk/embedding-sdk-api-extractor.json`

Reply:
Super awesome, thanks!